ampd/src/event_sub/mod.rs

```diff
@@ -79,6 +102,8 @@ impl Default for Config {
             poll_interval: Duration::from_secs(1),
             retry_delay: Duration::from_secs(3),
             retry_max_attempts: 3,
+            stream_buffer_size: 100000,
+            delay: Duration::from_secs(1),
```

Comment:
why are these default functions needed? I understand that the default for a missing value from deserialization would be 0 otherwise, I'm just curious why for these two values specifically and none of the others.
If we keep the default functions, they should also be used in the Default implementation to avoid drift